client/my-sites/post-share/style.scss

```diff
@@ -0,0 +1,140 @@
+.posts__list .post-share__wrapper {
+	margin: 0;
+	animation: appear .3s ease-in-out;
+	border-top: solid 1px transparentize( lighten( $gray, 20% ), .5 );
+
+	.post-share__title {
```

Comment:
Does all of this (and below) need to be nested under `.posts__list .post-share__wrapper `?